portsyncd/portsyncd.cpp

```diff
@@ -314,3 +336,17 @@ void handlePortConfig(ProducerStateTable &p, map<string, KeyOpFieldsValuesTuple>
         }
     }
 }
+
+void checkPortInitDone(DBConnector *appl_db)
```

Comment:
>checkPortInitDone [](start = 5, length = 17)

I think it is not necessary to check port table. For warm reboot, ignore all code related initialization, and just keep processing netlink messages or port config events.

Reply:
Yes, the check is optional.  It was designed to wok with the two different approaches, coupled/decoupled  portsyncd and portorch. Will remove it.